sphinx_scality/static/js/searchtools.js

```diff
@@ -0,0 +1,508 @@
+        // same score: sort alphabetically
+        left = a[1].toLowerCase();
+        right = b[1].toLowerCase();
+        return (left > right) ? -1 : ((left < right) ? 1 : 0);
+      }
+    });
+
+    // for debugging
+    //Search.lastresults = results.slice();  // a copy
+    //console.info('search results:', Search.lastresults);
+
+    // print the results
+    var resultCount = results.length;
+    function displayNextItem() {
+      // results left, load the summary and display it
+      if (results.length) {
+        var item = results.pop();
+        var listItem = $('<li style="display:none"></li>');
+        if (DOCUMENTATION_OPTIONS.FILE_SUFFIX === '') {
+          // dirhtml builder
+          var dirname = item[0] + '/';
+          if (dirname.match(/\/index\/$/)) {
+            dirname = dirname.substring(0, dirname.length-6);
+          } else if (dirname == 'index/') {
+            dirname = '';
+          }
+          listItem.append($('<a/>').attr('href',
+            DOCUMENTATION_OPTIONS.URL_ROOT + dirname +
+            highlightstring + item[2]).html(item[1]));
+        } else {
+          // normal html builders
+          listItem.append($('<a/>').attr('href',
+            item[0] + DOCUMENTATION_OPTIONS.FILE_SUFFIX +
+            highlightstring + item[2]).html(item[1]));
+        }
+        if (item[3]) {
+          listItem.append($('<span> (' + item[3] + ')</span>'));
+          Search.output.append(listItem);
+          listItem.slideDown(5, function() {
+            displayNextItem();
+          });
+        } else if (DOCUMENTATION_OPTIONS.HAS_SOURCE) {
+          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + item[0] + DOCUMENTATION_OPTIONS.FILE_SUFFIX,
+                  dataType: "text",
+                  complete: function(jqxhr, textstatus) {
+                    var data = jqxhr.responseText;
+                    if (data !== '' && data !== undefined) {
+                      listItem.append(Search.makeSearchSummary(data, searchterms, hlterms));
+                    }
+                    Search.output.append(listItem);
+                    listItem.slideDown(5, function() {
+                      displayNextItem();
+                    });
+                  }});
+        } else {
+          // no source available, just display title
+          Search.output.append(listItem);
+          listItem.slideDown(5, function() {
+            displayNextItem();
+          });
+        }
+      }
+      // search finished, update title and status message
+      else {
+        Search.stopPulse();
+        Search.title.text(_('Results'));
+        if (!resultCount)
+          Search.status.html(_('Your search did not match any documents. Try browsing the \
+          <a href="https://support.scality.com/hc/en-us">Knowledge Base</a> or contact Scality Support.'));
+        else
+            Search.status.html(_('Found %s page(s) matching the term(s) "<span class="term">%t</span>". Still not what you are\
```

Comment:
IIRC, those are the lines you changed, and they are slightly too long (we prefer to maintain a maximum of 80 columns, for easy reading on split/vertical screens).

Also, and it might have been like this before, we could improve the syntax used for clarity (assumes you defined a constant somewhere called `KB_LINK`, could prove useful since you use it two lines above):

```javascript
Search.status.html(_(`
  Found ${resultCount} page(s) matching the term(s) "<span class="term">${searchterms}</span>".
  Still not what you're looking for? Try browsing the <a href="${KB_LINK}">Knowledge Base</a>
  or contact Scality Support.
`));
```

Reply:
the ${...} do not work (whether $searchterms or $KB_LINK)
Keeping what we already had, but improving the variables (%searchterm instead of %t)